WordPressKit/WordPressComRestApi.swift

```diff
@@ -154,6 +156,14 @@ open class WordPressComRestApi: NSObject {
         uploadSessionManager.session.invalidateAndCancel()
     }
 
+    public func authTokenEquals(token: String) -> Bool {
+        return token == oAuthToken
+    }
+
+    @objc func setInvalidTokenHandler(_ handler: @escaping () -> Void) {
```

Comment:
While this might benefit from a `typealias` to better characterize the intent of the completion handler, this works fine as-is.